app/layout.js

```diff
@@ -14,6 +15,21 @@ export const metadata = {
 };
 
 export default function RootLayout({ children }) {
+  const router = usePathname();
+  if (router == "/registration") {
```

Comment:
You are checking router here, but this kind of thing is already implemented in `layout` > `navigation.js` file
Please implement this logic there.
It would be more accurate to keep all same functionality in one place.